changelogs/fragments/3401-nmcli-needs-type.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - "nmcli - the option ``routing_rules4`` can now specified as a list of strings, instead of as a single string (https://github.com/ansible-collections/community.general/issues/3401)."
```

Comment:
```suggestion
  - "nmcli - the option ``routing_rules4`` can now be specified as a list of strings, instead of as a single string (https://github.com/ansible-collections/community.general/issues/3401)."
```